src/schedule.rs

```diff
@@ -635,7 +699,7 @@ mod test {
             .checked_add_signed(chrono::Duration::hours(1)) // puts it in the middle of the DST transition
             .unwrap();
         let schedule = Schedule::from_str("* * * * * Sat,Sun *").unwrap();
-        let prev = schedule.after(&dt).rev().next().unwrap();
+        let prev = schedule.after(&dt).rev().skip(1).next().unwrap();
```

Comment:
Since we're now including both hours at daylight savings time change in the fall, we need to skip over one of the added times for the below assert to pass.